src/pixel/state.ts

```diff
@@ -50,7 +51,12 @@ const paramExtractors: ((state: State) => [string, string][])[] = [
   ifDefined('privacyMode', privacyMode => asStringParamWhen('nb', privacyMode ? 1 : 0, v => v === 1)),
   ifDefined('gdprConsent', gdprConsentString => asStringParam('gdpr_consent', gdprConsentString)),
   ifDefined('referrer', referrer => asStringParam('refr', referrer)),
-  ifDefined('contextElements', contextElements => asStringParam('c', contextElements))
+  ifDefined('contextElements', contextElements => asStringParam('c', contextElements)),
+  state => asStringParamWhen('pu_rp', isPageRemoved(state.pageUrl, state) ? 1 : 0, v => v === 1),
```

Comment:
I would combine both of these and the url one so we are not doing duplicate work

Reply:
Thanks, I just realized I can do that. At first I thought it wouldn't work properly. Will change it